### DISCUSSION TITLE
support

### DISCUSSION THREAD

**Discussion by evaelfie0** [Genymobile/feedback#44] [Genymotion Desktop] [answered]
If a device could be set to automatically boot when Genymotion Desktop launches, that would be wonderful. So click this [website](https://ddrivingapk.com/dr-driving-mod-apk-ios/)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @evaelfie0,

You can use gmtool CLI to write scripts to start/stop a device. For more details about gmtool, see [gmtool user guide](https://docs.genymotion.com/desktop/06_GMTool/#gmtool-admin).
